nvidia/gmsl_camera/camera_driver.rst

```diff
@@ -3,4 +3,76 @@
 Camera Driver Usage
```

Comment:
"Camera Driver Installation" might be better.

Reply:
Yes